velox/dwio/parquet/reader/ParquetData.h

```diff
@@ -199,6 +217,13 @@ class ParquetData : public dwio::common::FormatData {
   // ahead of first use, not at construction.
   std::vector<std::unique_ptr<dwio::common::SeekableInputStream>> streams_;
 
+  bool isParquetBloomFilterEnabled_;
+  // RowGroup+Column to BloomFilter map
+  std::unordered_map<
+      uint32_t,
+      std::unordered_map<uint32_t, std::shared_ptr<BloomFilter>>>
```

Comment:
In my understanding, we don't need to use a 2-level map here. Each column reader own's its ParquetData object by itself, therefore it doesn't need to store the column index. Would you please try just store <RowGroupId, BloomFilter>?